packages/gatsby-theme-newrelic/src/components/GlobalSearch.js

```diff
@@ -0,0 +1,215 @@
+      return next;
+    });
+
+  useEffect(() => {
+    setSelected(null);
+  }, [query]);
+
+  const searchRef = useRef(null);
+  const { t } = useThemeTranslation();
+
+  useKeyPress('/', (e) => {
+    // prevent quick search from opening in Firefox
+    e.preventDefault();
+    // rAF prevents `/` from being typed in input after focusing
+    requestAnimationFrame(() => searchRef.current?.focus());
+  });
+
+  const showSearchDropdown = query.length > 1 && open;
+
+  return (
+    <>
+      <SearchInput
+        onClear={() => {
+          setQuery('');
+          setOpen(false);
+          onClose();
+        }}
+        onFocus={() => setOpen(true)}
+        onMove={(direction) => {
+          if (direction === 'prev') {
+            moveUp();
+          } else {
+            moveDown();
+          }
+        }}
+        onSubmit={(value) => {
+          if (value.length < 2) return;
+          if (selected != null) {
+            if (selected > recentQueries.length - 1) {
+              const position = selected - recentQueries.length;
+              const selectedResult = results[position];
+              saveSearch(value);
+              addPageAction({
+                eventName: 'swiftypeSearchResult',
+                category: 'GlobalSearch',
+                path: location.pathname,
+                resultCount: results.length,
+                position,
+                searchTerm: query,
+                searchType: 'globalSearch',
+                url: selectedResult.url,
+              });
+              navigate(selectedResult.url);
+            } else {
+              const position = selected;
+              const recentQuery = recentQueries[position];
+              addPageAction({
+                eventName: 'savedQuerySearch',
+                category: 'GlobalSearch',
+                path: location.pathname,
+                searchTerm: recentQuery,
+                position,
+                searchType: 'globalSearch',
+              });
+              navigate(`/search-results?query=${recentQuery}&page=1`);
+            }
+          } else {
+            saveSearch(value);
+            addPageAction({
+              eventName: 'swiftypeSearchInput',
+              category: 'GlobalSearch',
+              path: location.pathname,
+              resultCount: results.length,
+              searchTerm: query,
+              searchType: 'globalSearch',
+            });
+            navigate(`/search-results?query=${value}&page=1`);
+          }
+        }}
+        placeholder={t('searchInput.placeholder')}
+        ref={searchRef}
+        setValue={setQuery}
+        size={SearchInput.SIZE.MEDIUM}
+        value={query}
+        css={css`
+          --icon-size: 1.5rem;
+          width: var(--search-width);
+
+          svg {
+            width: 1rem;
+            height: 1rem;
+          }
+
+          input {
+            border: none;
+            height: 40px;
+          }
+
+          // TODO: use right bp here
```

Comment:
Seems like the right bp, is there anything that needs to get updated here?

Reply:
hmm well technically this breakpoint is specified in the theme config. but this is the right breakpoint from the designs and i don't think it would be reliable for this to be configurable. i'll remove this todo